embedded-graphics-core/src/lib.rs

```diff
@@ -0,0 +1,84 @@
+//! Embedded Graphics Core
+//!
+//! DOC TODO
```

Comment:
@rfuest I could use a hand here. Bullet points are fine, but I'm drawing a blank at the moment.

Reply:
I had a cup of tea and the caffeine whispered sweet nothings into my brain. I've written some filler. Not sure if it's really in the right direction, but it's something we can work on.